src/document/common/common-definitions.xsl

```diff
@@ -152,6 +153,7 @@
             </div>
             <xsl:where-populated>
                 <div class="body">
+                    <xsl:apply-templates select="$definition/@deprecated"/>
```

Comment:
Nice! adding to the default traversal by selecting and processing the attribute.